packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -181,59 +198,24 @@ export default class SaytDriverPlugin implements Plugin {
 
   /**
    * Extracts query and products from the given response.
+   * Calls `this.transformProduct` on each product found in the response.
+   * Filters out any products that map to a falsy value.
    *
    * @param response An object containing the original search response.
    * @returns An object containing an array of valid simplified products and the original response.
    */
-  searchCallback(searchResponse: Results): SaytProductsResponsePayload {
+  searchCallback(searchResponse: Results): SaytProductsResponsePayload<P> {
     const { records } = searchResponse;
-    const mappedRecords = records.map((record) => {
```

Comment:
The below code (the callback passed to `map()`) now exists in the View Layer demo.